src/styles/utility/mixins.scss

```
@@ -0,0 +1,4 @@
+@mixin star-size {
+  height: 16px;
+  width: 16px;
+}
```

Create mixin that can get values for creating square and use it for logo and for stars
```suggestion
@mixin square($size) {
  height: $size;
  width: $size;
}
```